js/main.js

```diff
@@ -1,330 +0,0 @@
-// main.js
-console.log('main.js')
-
-// global card counter
-let cardNum = 0;
-
-//global array, keep track of zip codes
-zipArray = [];
-
-// Set const to body and assign id with "bodyid"
-let htmlBody = document.querySelector('body');
-htmlBody.setAttribute('id','bodyid');
-
-// Create const mainDiv holds all html elements for weather app
-const mainDiv = document.createElement('div');
-mainDiv.setAttribute('id','main');
-htmlBody.appendChild(mainDiv);
-
-// Create a set of headerDiv and bodyDiv for our content
-// Use as reference for when appending childs
-const headerDiv = document.createElement('div');
-headerDiv.setAttribute('id', 'headerDiv');
-htmlBody.appendChild(headerDiv);
-const bodyDiv = document.createElement('div');
-bodyDiv.setAttribute('id', 'bodyDiv');
-htmlBody.appendChild(bodyDiv);
-
-
-// testing append on body
-// htmlBody.appendChild(mainDiv)
-// htmlBody.appendChild(titleTextElem)
-
-// Function for creating html elements for DOM with JS
-// Create a func with Params passing
-// html element name and name of the id
-// then append element to a parent Param passed in the function
-function createElem(htmlElem, idName, parentElem, className = null){
-    const parent = document.getElementById(parentElem)
-    const makeElem = document.createElement(htmlElem);
-    makeElem.setAttribute('id', idName); 
-    if (className !== null)
-    makeElem.setAttribute('class', className)
-    parent.appendChild(makeElem);
-
-    //return parentElem;
-
-}
-
-//createElem('h1', 'anotherh1', htmlBody);
-// headerDivElem array values/ order
-// key: html Element, ID Name, Parent Name, Class Name(Using Bootstrap)
-const headerDivElem = {
-    card: ['div','card','headerDiv','card text-center'],
-    //cardHeader: ['div','cardHeader','card','card-header'],
-    cardBody: ['div','cardBody','card','card-body'],
-    cardTitle: ['h3','cardTitle','cardBody','card-title'],
-    cardText: ['p','cardText','cardBody','card-text'],
-
-    cardRow: ['div','cardRow','cardBody','row-6'],
-    cardForm: ['form','cardForm','cardRow','row justify-content-center'],
-    colDiv1: ['div','colDiv1','cardForm','col-6 col-md-3'],
-    cardInput: ['input','cardInput','colDiv1','form-control form-control-m'],
-    colDiv2: ['div','colDiv2','cardForm','col-6 col-md-3'],
-    cardButton: ['button','cardButton','colDiv2','btn btn-primary'],
-
-
-    //cardFooter: ['div','cardFooter','card','card-footer text-muted']
-
-}
```

Comment:
clean up all the extra lines (don't want them unless needs to be broken up for grouping. Also delete any commented out stuff throughout the doc.